pyproject.toml

```diff
@@ -33,6 +33,8 @@ psycopg2 = "^2.9.6"
 sentry-sdk = "^1.19.1"
 django-storages = "^1.13.2"
 django-anymail = "^9.1"
+django-elasticsearch-dsl-drf = "^0.22.5"
+django-opensearch-dsl = "^0.5.0"
```

Comment:
we don't need to explicitly specify it, since it's a dependency of `ango-elasticsearch-dsl-drf`, also we should add it  in the applications and make sure it's setup properly :)

Reply:
I didn't specify it, poetry locked versions when doing `poetry add`